.github/workflows/build.yml

```diff
@@ -91,6 +91,50 @@ jobs:
           name: abi-data
           path: ./Doc/data/*.abi
 
+  check_autoconf_regen:
+    name: 'Check if Autoconf files are up to date'
+    # Don't use ubuntu-latest but a specific version to make the job
+    # reproducible: to get the same tools versions (autoconf, aclocal, ...)
+    runs-on: ubuntu-24.04
```

Comment:
ditto: Is it okay to use 24.04 at 3.12?

Reply:
Yes, it'll be easier to maintain to have the same on each branch.